docs/instance_groups.md

```diff
@@ -83,7 +84,7 @@ The procedure to resize the root volume works the same way:
 
 For example, to set up a 200GB gp2 root volume, your InstanceGroup spec might look like:
 
-```
+```YAML
 metadata:
   creationTimestamp: "2016-07-11T04:14:00Z"
```

Comment:
I think these creation timestamps can be removed also. I don't see much value to have them here.

Reply:
Good point, only seems likely to confuse people. Addressed.